templates/toil-infra-essentials.yaml

```diff
@@ -71,9 +75,8 @@ Resources:
             Principal:
               AWS:
                 - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
-                - !ImportValue us-east-1-rna-seq-reprocessing-instance-role-v001-ToilClusterRoleArn
-                - !If [IsSandbox, !ImportValue us-east-1-rna-seq-reprocessing-role-dev-ToilClusterRoleArn, !Ref "AWS::NoValue"]
                 - !FindInMap [AdminRoleArns, !Ref "AWS::AccountId", Arn]
+                - !If [HasToilClusterRoleArn, !Ref ToilClusterRoleArn, !Ref "AWS::NoValue"]
```

Comment:
so what happens when `HasToilClusterRoleArn` is not passed in?

Reply:
HasToilClusterRoleArn is a condition, not a parameter.

Reply:
so the default for ToilClusterRoleArn  is set to empty.  if it's nothing is set then nothing gets set here.  will this system still work if ToilClusterRoleArn is not set here?

Reply:
I can either make this work as a whole system (which is preferable to me) or make this work so that it's easier to change one stack at a time -- which is my understanding of what IT prefers. I am doing the thing where ToilClusterRoleArn can be empty to make it easier to work on one piece of the system at a time.

Reply:
lint_stack is *still* commented out in .travis.yml because of the one-stack-changed-at-a-time limitation.